### PR TITLE
Feature/add webrtc recipe

### PR DESCRIPTION
**Pull request by eramirez392**

### CODE REVIEW

File: recipes-multimedia/usrsctp/usrsctp_0.9.3.bb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Usrsctp"
+DESCRIPTION = "Userland SCTP stack supporting FreeBSD, Linux, Mac OS X and Windows"
+SECTION = "multimedia"
+LICENSE = "BSD"
+
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=ffcf846341f3856d79a483eafa18e2a5"
+
+DEPENDS = ""
```
Comment (rrcarlosrodriguez) [inline recipes-multimedia/usrsctp/usrsctp_0.9.3.bb]: Is it necessary to set DEPENDS if there is nothing there?

File: recipes-multimedia/usrsctp/usrsctp_0.9.3.bb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+SRCBRANCH ?= "master"
+SRCREV = "a9eb516d23d3eec6f2779d1b8965df01996a9836"
+SRC_URI = " \
+    git://git@github.com/sctplab/usrsctp.git;protocol=http;branch=${SRCBRANCH} \
+    file://0001-usrsctp-yocto-compability.patch"
+
+S = "${WORKDIR}/git"
+
+PACKAGECONFIG_CONFARGS = ""
```
Comment (rrcarlosrodriguez) [inline recipes-multimedia/usrsctp/usrsctp_0.9.3.bb]: Is it necessary?

File: recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "
+
+S = "${WORKDIR}/git"
+
+PACKAGECONFIG_CONFARGS = " \
+	--disable-gtk-doc \
+"
+FILES_${PN} += "\
+	${libdir}/gstreamer-1.0/libgstnice010.la \
+	${libdir}/gstreamer-1.0/libgstnice010.so \
```
Comment (rrcarlosrodriguez) [inline recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb]: Why is libgstnice here?

**Review by rrcarlosrodriguez — CHANGES_REQUESTED** [review]
Some questions

File: recipes-multimedia/usrsctp/usrsctp_0.9.3.bb (hunk 1 above)
Comment (eramirez392) [reply recipes-multimedia/usrsctp/usrsctp_0.9.3.bb]: It's not. I'll remove that.

File: recipes-multimedia/usrsctp/usrsctp_0.9.3.bb (hunk 2 above)
Comment (eramirez392) [reply recipes-multimedia/usrsctp/usrsctp_0.9.3.bb]: It's not. I'll remove that.

File: recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb (hunk 3 above)
Comment (eramirez392) [reply recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb]: When compiling the recipe, some warnings appear related to those paths were missing when installing the libraries

File: README.md [hunk 4]
```diff
@@ -28,4 +28,22 @@ export MACHINE=zcu106-zynqmp
...
+
+## Building Gst-WebRTC (Private)
+
+-GstWebRTC is an add-on to RidgeRun's professional SDK. You can purchase GstWebRTC with full source code, from:
+    https://www.ridgerun.com/store/GSTWebRTC-p74337777
+
+-Once you have access to the repository, please open gst-webrtc_1.8.1.bb in **$YOCTO_DIRECTORY/sources/meta-ridgerun/recipes-multimedia/gstreamer/**
+
+-Modify SRC_URI with the correct gst-webrtc URL by changing **<Customer-Directory>** with your own.
+
```
Comment (josejich) [inline README.md:41]: Nice

File: recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+DEPENDS = "gstreamer1.0 gstreamer1.0-plugins-base gstreamer1.0-plugins-bad json-glib libsoup-2.4 libnice autoconf libtool glib-2.0 openssl readline libsrtp usrsctp openwebrtc-gst-plugins"
+
+SRCBRANCH ?= "master"
+
+SRCREV_base = "52713a36f056d99e10e95de556eb164d9340ff9f"
+SRCREV_common = "b64f03f6090245624608beb5d2fff335e23a01c0"
+
+SRC_URI = " \ 
+    git://git@gitlab.com/RidgeRun/orders/<Customer-Directory>/gst-webrtc.git;protocol=ssh;branch=${SRCBRANCH};name=base \
```
Comment (josejich) [inline recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb:17]: nice again!

File: recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    "
+
+S = "${WORKDIR}/git"
+
+PACKAGECONFIG_CONFARGS = " \
+	--disable-gtk-doc \
+"
+FILES_${PN} += "\
+	${libdir}/gstreamer-1.0/libgstwebrtc.so \
+	${libdir}/gstreamer-1.0/libgstwebrtc.la \
```
Comment (josejich) [inline recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb:28]: Did you tested it? I think that we might need to install a couple of libraries more like basesignaler and pubnubsignaler.

Comment (eramirez392) [reply recipes-multimedia/gstreamer/gst-webrtc_1.8.1.bb:28]: Yes, I checked and the other libraries were installed correctly.

**Review by josejich — APPROVED** [review]

**Review by michaelgruner — APPROVED** [review]

**Pull request merged**